### PR TITLE
Update Homepage

### PR DESCRIPTION
**Pull request by specterflare**
Oxford comma for the win.

### CODE REVIEW

**Comment by Soitora** [comment]
Why did you rename the action button back to "Getting started"?

File: src/README.md [hunk 1]
```diff
@@ -3,17 +3,17 @@ home: true
...
   - title: Extensions
-    details: Online reading from sources such as MangaDex, MangaKakalot and hundreds more.
+    details: Online and offline reading from sources like MangaDex, MangaKakalot, and hundreds more
     image: Browse
   - title: Customization
-    details: A configurable reader with multiple viewers, reading directions and other settings.
+    details: A configurable reader with multiple reading modes, custom color filters, and other settings
     image: Manga
-footer: Apache Licensed | Copyright © 2015-present Javier Tomás
+footer: Apache Licensed | Copyright © 2015-present by Javier Tomás
```
Comment (arkon) [inline src/README.md:18]: It's typically worded like it was before.

Comment (specterflare) [reply src/README.md:18]: Oh, OK.

**Comment by specterflare** [comment]
My thought process was that if someone is first coming to the site, the first thing they are going to do is download Tachiyomi, and the second thing they will want to do is learn how to set it up. I feel like "Get started" is much more direct in its intention than "User guide", and communicated more effectivity where the button will land you.

**Comment by specterflare** [comment]
I don't know how to request a review.

**Comment by Soitora** [comment]
I get your intentions for "Get started" but I still prefer the current "User guide", other than that I think the PR is fine

**Comment by specterflare** [comment]
OK, done.

**Pull request merged**